### ISSUE TITLE
Amazon VPC Flow Logs now support 1-minute aggregation intervals

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#11927]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

> You can now capture and aggregate your Amazon Virtual Private Cloud (Amazon VPC) flow logs at shorter intervals of up to 1 minute, giving you quicker visibility into your network traffic flows.

### New or Affected Resource(s)

* [`aws_flow_log`](https://www.terraform.io/docs/providers/aws/r/flow_log.html)

### Potential Terraform Configuration

```hcl
resource "aws_flow_log" "example" {
  max_aggregation_interval = 60
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2020/02/amazon-vpc-flow-logs-support-1-min-aggregation-intervals/).

Requires [AWS SDK v1.28.10](https://github.com/aws/aws-sdk-go/releases/tag/v1.28.10):
* https://github.com/terraform-providers/terraform-provider-aws/pull/11893

**Comment by bflad** [comment]
Support for this has been merged and will release with version 2.55.0 of the Terraform AWS Provider, likely later today. Thanks to @ewbankkit and @ajorpheus for the implementation. 👍

**Comment by kevincantu** [comment]
😺
